packages/@lwc/features/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@lwc/features",
```

Comment:
In the current state of the PR, this package can't be used outside the monorepo. I think it would be made private for now.